src/main/msp/msp_protocol_v2_betaflight.h

```diff
@@ -34,3 +34,4 @@
 #define MSP2TEXT_RATE_PROFILE_NAME               4
 #define MSP2TEXT_BUILDKEY                        5
 #define MSP2TEXT_RELEASENAME                     6
+#define MSP2TEXT_CUSTOM_MSG                      7  //add for msp set custom message
```

Comment:
Define first message here + define last allocated or maximum custom message count (maybe 8, not allocated buffer space)

Reply:
Thank you. This is indeed easier to extend more messages. I'm trying to modify.

Reply:
I have modified the code and extend to support 8 messages.